documenting.rst

```diff
@@ -1679,8 +1679,8 @@ in production, other are work in progress:
 .. _mail_it: https://mail.python.org/pipermail/doc-sig/2019-April/004114.html
 .. _mail_lt: https://mail.python.org/pipermail/doc-sig/2019-July/004138.html
 .. _mail_ru: https://mail.python.org/pipermail/doc-sig/2019-May/004131.html
-.. _tx_pl: https://www.transifex.com/python-doc/python-newest/language/pl/
-.. _tx_zh_cn: https://www.transifex.com/python-doc/python-newest/language/
+.. _tx_pl: https://www.transifex.com/python-doc/python-newest/
+.. _tx_zh_cn: https://www.transifex.com/python-doc/python-newest/
```

Comment:
Does this look correct, @JulienPalard? Are there better urls for these?

Reply:
This is complicated. Transifex give 404 when someone is not logged (I don't like it).

So https://www.transifex.com/python-doc/python-newest/language/pl/ and https://www.transifex.com/python-doc/python-newest/language/zh_CN/ are the real URLs, which only work for logged in users. Maybe  https://www.transifex.com/python-doc/python-newest/ is more friendly, as I don't expect much people to already be logged in on transifex when clicking the link.